subprojects/core/src/integTest/groovy/org/gradle/configuration/ExecuteUserLifecycleListenerBuildOperationIntegrationTest.groovy

```diff
@@ -375,15 +375,13 @@ class ExecuteUserLifecycleListenerBuildOperationIntegrationTest extends Abstract
         ]
         def addProjectListeners = { String source, String target = null ->
             """
-            ${target == null ? '' : "project('$target') { project ->"}
```

Comment:
This calls `DefaultProject.evaluate()` That happens in execution time. It is exactly what this PR disables.